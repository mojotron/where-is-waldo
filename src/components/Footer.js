import "../styles/Footer.css";

const Footer = () => {
  return (
    <footer className="Footer">
      <h3 className="Footer__info">
        created by{" "}
        <a
          className="Footer__info__link"
          href="https://github.com/mojotron/where-is-waldo"
          target="_blank"
          rel="noopener noreferrer"
        >
          @mojotron
        </a>
      </h3>
    </footer>
  );
};

export default Footer;
