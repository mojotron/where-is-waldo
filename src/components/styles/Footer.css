.Footer {
  color: var(--red);
  padding: 1rem 0 5rem;
}
.Footer__info__link {
  opacity: 0.5;
}
.Footer__info__link:hover,
.Footer__info__link:active {
  color: #74787c;
  opacity: 1;
}
