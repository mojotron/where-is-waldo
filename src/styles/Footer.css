.Footer {
  height: 10vh;
  background-color: var(--red);
}
