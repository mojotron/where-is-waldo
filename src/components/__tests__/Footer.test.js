import { render, screen } from "@testing-library/react";
import Footer from "../Footer";

describe("Footer component", () => {
  test("renders correct elements", () => {
    const { component } = render(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
