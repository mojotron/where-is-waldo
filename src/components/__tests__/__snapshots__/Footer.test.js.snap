// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correct elements 1`] = `undefined`;
